pkgs/misc/themes/arc/default.nix

```diff
@@ -1,43 +1,55 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
+{ stdenv, fetchFromGitHub, sassc, autoreconfHook, pkgconfig, gnome3
+, gnome-themes-standard, gtk-engine-murrine, optipng, inkscape}:
 
 let
-  # treat versions newer than 3.22 as 3.22
-  gnomeVersion = if stdenv.lib.versionOlder "3.22" gnome3.version then "3.22" else gnome3.version;
+  # per README: For GNOME 3.24 and 3.26, use --with-gnome=3.22
+  gnomeVersion = if gnome3.version == "3.24" || gnome3.version == "3.26" then "3.22" else gnome3.version;
   pname = "arc-theme";
 in
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
-  version = "2017-05-12";
+  version = "2018-07-13";
 
   src = fetchFromGitHub {
-    owner  = "horst3180";
+    owner  = "NicoHood";
     repo   = pname;
-    rev    = "8290cb813f157a22e64ae58ac3dfb5983b0416e6";
-    sha256 = "1lxiw5iq9n62xzs0fks572c5vkz202jigndxaankxb44wcgn9zyf";
+    rev    = "be675c9e8c853dd4a20f071b408158b0d871f1a4";
+    sha256 = "1kkfnkiih0i3pds5mgd1v9lrdrp4nh4hk42mw7sa4fpbjff4jh6j";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig ];
-  buildInputs = [ gnome3.gtk ];
+  preBuild = ''
+    # Shut up inkscape's warnings
+    export HOME="$NIX_BUILD_ROOT"
+  '';
 
-  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig sassc optipng inkscape ];
+  buildInputs = [ gnome3.gtk ];
 
-  preferLocalBuild = true;
+  propagatedUserEnvPkgs = [ gnome-themes-standard gtk-engine-murrine ];
+
+  postPatch = ''
+    find . -name render-assets.sh |
+    while read filename
+    do
+      substituteInPlace "$filename" \
+        --replace "/usr/bin/inkscape" "${inkscape.out}/bin/inkscape" \
+        --replace "/usr/bin/optipng" "${optipng.out}/bin/optipng"
+    done
+    patchShebangs .
+  '';
 
   configureFlags = [ "--disable-unity" "--with-gnome=${gnomeVersion}" ];
 
   postInstall = ''
-    mkdir -p $out/share/plank/themes
-    cp -r extra/*-Plank $out/share/plank/themes
-    install -Dm644 -t $out/share/doc/${pname}/Chrome extra/Chrome/*.crx
     install -Dm644 -t $out/share/doc/${pname}        AUTHORS *.md
   '';
 
   meta = with stdenv.lib; {
-    description = "A flat theme with transparent elements for GTK 3, GTK 2 and Gnome-Shell";
-    homepage    = https://github.com/horst3180/arc-theme;
+    description = "[New upstream] A flat theme with transparent elements";
```

Comment:
Maybe it is better not to change the description. `[New upstream]` may not be clear enough to convey the fact that we are using a fork of the now unmaintained package.